libkbfs/mdserver_remote.go

```diff
@@ -1096,6 +1128,11 @@ func (md *MDServerRemote) GetKeyBundles(ctx context.Context,
 	return wkb, rkb, nil
 }
 
+// FastForwardBackoff implements the MDServer interface for MDServerRemote.
+func (md *MDServerRemote) FastForwardBackoff() {
```

Comment:
This function should take `md.connMu` to avoid races with `initNewConnection()`.